### ISSUE TITLE
[MacOS] 100% CPU in Chrome 67

### ISSUE THREAD

**Issue by panpiotrs** [Tampermonkey/tampermonkey#552]
Hello,

after browser update the extension eats constantly 100% CPU. Restarting browser and extension has no effect on this (the same when all scripts are disabled), browser/extension launch and right away it skyrockets to 99-100%. I need this really badly, any ideas what can I do?

Chrome 67.0.3396.62, MacOS 10.13.4

**Comment by panpiotrs** [comment]
I reinstalled the extension and it was completely fine, then:
1) I added my script without changing any settings (I pasted my code and saved), script is working, extension behave like it should, everything is perfect
2) I changed language from "Browser default" to "English" and it goes into 100% mode again...I reinstalled it again, added my script, everything was fine and I imported my old settings (exported today before first reinstall) and CPU 100% again. I reinstalled again and changed config mode from Novice to Advanced - no problems with CPU.

I don't have time to play with this more, but now it looks like changing the language makes it go crazy. It's weird because I had "Browser default" in my settings before, so probably there are other settings which also trigger that.
